GameServerLib/Logging/Logger.cs

```diff
@@ -16,56 +16,37 @@ public Logger(ILog log)
         public void Debug(object message)
         {
             if (_log.IsDebugEnabled)
-                _log.Debug(AppendCallingMethodName(message?.ToString() ?? string.Empty));
+                _log.Debug(message?.ToString() ?? string.Empty);
         }
 
         public void Info(object message)
         {
             if (_log.IsInfoEnabled)
-                _log.Info(AppendCallingMethodName(message?.ToString() ?? string.Empty));
+                _log.Info(message?.ToString() ?? string.Empty);
         }
 
         public void Warning(object message)
         {
             if (_log.IsWarnEnabled)
-                _log.Warn(AppendCallingMethodName(message?.ToString() ?? string.Empty));
+                _log.Warn(message?.ToString() ?? string.Empty);
         }
 
         public void Error(object message)
         {
             if (_log.IsErrorEnabled)
-                _log.Error(AppendCallingMethodName(message?.ToString() ?? string.Empty));
+                _log.Error(message?.ToString() ?? string.Empty);
         }
 
         public void Error(Exception ex)
         {
             if (_log.IsErrorEnabled)
-                _log.Error(AppendCallingMethodName(ex?.Message ?? string.Empty), ex);
+                _log.Error(ex?.Message ?? string.Empty, ex);
         }
 
         public void Error(object message, Exception ex)
         {
             if (_log.IsErrorEnabled)
-                _log.Error(AppendCallingMethodName(message?.ToString() ?? string.Empty), ex);
-        }
-
-        private string AppendCallingMethodName(string message)
```

Comment:
This change removes the information from what method the logger was called. To remove the redundant information from log you can alter app.config and remove the [%logger] part in conversionPattern element. Or CallerMemberName might work.

Reply:
The logger has this information, I need to update the ConsoleAppender to display it or make a custom ConsoleAppender that displays it.
[Not yet resolved]

Reply:
Resolved